From 3f0009f4c4b98c9380089d42cf229d603711e877 Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Sun, 29 Mar 2020 14:49:41 -0700
Subject: [PATCH] Mention $status_format %B expando in background edit section.

It's added to the default, but some users will have customized the
value and may miss the new expando.
---
 doc/manual.xml.head | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/manual.xml.head b/doc/manual.xml.head
index c73206bb..ed471221 100644
--- a/doc/manual.xml.head
+++ b/doc/manual.xml.head
@@ -1830,7 +1830,9 @@ fo-table</literal> for details.
   To help prevent forgetting about backgrounded sessions, <link
   linkend="background-confirm-quit">$background_confirm_quit</link>
   will prompt before exiting, in addition to <link
-  linkend="quit">$quit</link>.
+  linkend="quit">$quit</link>.  Additionally, the <literal>%B</literal>
+  expando in <link linkend="status-format">$status_format</link>
+  displays the number of backgrounded compose sessions.
 </para>
 <para>
   Background editing is available for most, but not all, message
-- 
2.26.2

